modules/pages/client/components/Donate.component.js

```diff
@@ -0,0 +1,160 @@
+
+            <br />
+          </div>
+          <div className="col-xs-12 col-sm-6 col-md-5">
+            <p className="lead">
+              <em>
+                {t(
+                  "We're currently working on setting up ways to receive donations. Until that, this page gives you preliminary information about our funding model.",
+                )}
+              </em>
+            </p>
+            <hr />
+            <h2>{t('Ways to Give')}</h2>
+            <h3>{t('Credit/debit card')}</h3>
+            {t(
+              "We don't have a credit card transfer set up yet at this point.",
+            )}
+            <h3>{t('Bank transfer')}</h3>
+            {t("We don't have a bank account set up yet at this point.")}
+            <h3>Bitcoin</h3>
+            <Trans t={t} ns="pages">
+              <a href="/support">Contact us</a> if you want to donate through
+              Bitcoin.
+            </Trans>
+            <h3>PayPal</h3>
+            <Trans t={t} ns="pages">
+              <a href="/support">Contact us</a> if you want to donate through
+              PayPal.
+            </Trans>
+            <h3>{t('Referal programs')}</h3>
+            <h4>{t('Namecheap')}</h4>
+            <Trans t={t} ns="pages">
+              We use and recommend Namecheap for domains, partly because
+              they&apos;re a{' '}
+              <a href="https://www.namecheap.com/about/causes.aspx">
+                responsible company
+              </a>{' '}
+              and foremost they do a great job. Sign up via{' '}
+              <a href="https://www.namecheap.com/?aff=90636">this link</a> and
+              we&apos;ll get little bit in return.
+            </Trans>
+            <br />
+            <h4>DigitalOcean</h4>
+            <Trans t={t} ns="pages">
+              Our servers are hosted at reliable and easy to use DigitalOcean.
+              Sign up via{' '}
+              <a href="https://www.digitalocean.com/?refcode=6dc078966c9c">
+                this link
+              </a>{' '}
+              and when you start using their VPS service, we get 25$ to cover
+              our hosting, you get 10$.
+            </Trans>
+            <hr />
+            <p>
+              <small>
+                <Trans t={t} ns="pages">
+                  By donating, you agree to share your personal information with
+                  the Trustroots Foundation, the non-profit organisation running
+                  Trustroots.org, pursuant to our{' '}
+                  <a href="/donate/policy">donor policy</a>. Trustroots
+                  Foundation is located in the United Kingdom. We do not sell or
+                  trade your information to anyone.
+                </Trans>
+              </small>
+            </p>
+          </div>
+          <a
+            className="btn btn-xs btn-primary pull-right"
+            href="https://github.com/Trustroots/trustroots/edit/master/modules/pages/client/views/donate.client.view.html"
```

Comment:
The edit links are not valid any more as they are pointing to angular templates.

With i18n, editing is a bit more complex, so arguably we could remove all the `Edit this page` links? Whaddya you say @simison?

Reply:
Good to go away indeed :thumbsup:

Reply:
Alright, took that out, too.